Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.js

```diff
@@ -142,6 +165,87 @@ WI.FontVariationDetailsSectionRow = class FontVariationDetailsSectionRow extends
 
         return 1;
     }
+
+    _showValidity()
+    {
+        if (!this._hasValidValue)
+            this.warningMessage = WI.UIString("Axis value outside of supported range: %s – %s", "A warning that is shown in the Font Details Sidebar when the value for a variation axis is outside of the supported range of values").format(this._formatAxisValueAsString(this._minimumValue), this._formatAxisValueAsString(this._maximumValue));
+        else
+            this.warningMessage = null;
+    }
+
+    _handleValueBlurEvent(event)
```

Comment:
Maybe `_handleValueTextFieldBlurEvent` to make it clear this is for the value text field and not the value slider? Same for the other handlers below.

Reply:
Sounds good 👍

I also renamed `_inputRangeElement`  to `_valueSliderElement` for consistency and because 
`_handleInputRangeInput` just sounded funny 😃